frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -1024,7 +1027,73 @@ export function Experiment(): JSX.Element {
                     </Row>
                     <div className="experiment-result">
                         {experimentResults ? (
-                            <>
+                            experimentData?.parameters?.feature_flag_variants?.length || 0 > 4 ? (
```

Comment:
This is the main conditional for the experiment results view:

When less than 4 total variants, we stick to the old view

When mooar, we switch to table view

Reply:
A vertical table might be a lot easier to look at rather than a horizontal one ! 😃

Reply:
Oh, interesting, that might be true.
 
 I'd rather not change again, unless you think this is blocking? 😅 .

Reply:
It's probably true lol but if it's emergency though, non-blocking

Reply:
Like using a lemon table or one of our existing table views

Reply:
oh, that makes sense 😅 - should've started with that